roles/openshift_node/tasks/main.yml

```diff
@@ -133,11 +133,20 @@
   retries: 120
   delay: 1
   changed_when: false
-  when: openshift.common.is_containerized | bool
 
 - name: Start and enable node
   service: name={{ openshift.common.service_type }}-node enabled=yes state=started
   register: node_start_result
-
+  failed_when: false
+  
+- name: Sleep 10 seconds when starting the node failed
+  pause: secs=10
+  when: node_start_result.state != 'started'
+  
+- name: Try starting the node again
+  service: name={{ openshift.common.service_type }}-node enabled=yes state=started
+  register: node_start_result
+  when: node_start_result.state != 'started'
```

Comment:
Can we reliably use delegate_to now?  I ask because I think it would be required.  This role is running on the Node yet the only way I know of to call the Node healthz url is with the admin cert and key which is only on the Master.  I think I'd be happy if this PR simply had some comments for now and then we continue working to find a way to have systemd do the correct thing and not kill the Node on startup.